### ISSUE TITLE
not needed `autoload` @ `composer.json`

### ISSUE THREAD

**Issue by xerc** [mediatis/form_fieldnames#2]
https://github.com/mediatis/form_fieldnames/blob/8fa40d726643e66813930cb45ebffd5b1eed6879/composer.json#L26-L30

btw/ t3 EXT updates not working @ https://extensions.typo3.org/extension/form_fieldnames

**Comment by s-ude** [comment]
@xerc Good catch, that part seems to be unnecessary. Though I also do not see any downside in having it in the `composer.json`, which is why this will have a lower priority for us. I will make a note so that we don't forget about it in future updates.

Regarding the updates in the TER, this is a bit tricky. We have dropped our TER support temporarily because many of our packages have non-TYPO3-packages as dependencies, which is something that TER cannot handle. However, the T3 core team has started to scan packagist and list TYPO3 extensions without the authors having to upload the extension directly in TER. This is something that we want to use as well, though there are open questions left, that we have to address first.

**Comment by xerc** [comment]
thanks for the TER info ; for me it seem that there will be multible EXT which will be duplicated
ref. github.com/linawolf/form_double_opt_in/issues/25
